styles.css

```diff
@@ -264,17 +471,28 @@
 }
 
 .contactForm-section {
-  height: auto;
+  height: 900px;
   width: 340px;
   display: flex;
   flex-direction: column;
   align-self: center;
+  align-items: center;
   padding-top: 80px;
-  background-image: url("images/contact-shapes-mobile.png"), url("images/contact-background.png");
+  background-image: url("images/contact-shapes-mobile.png"), url("images/contact-background-mobile.png");
   background-repeat: no-repeat;
   background-color: white;
 }
```

Comment:
This section is not following the template design. The width is too narrow, make sure once you fix the width you fix the background image position and make sure the input field and the button are positioned and styled properly.

Please fix this section's design in the mobile version as well.